preferences/src/main/java/dev/jdtech/jellyfin/AppPreferences.kt

```diff
@@ -34,6 +34,7 @@ constructor(
     val theme get() = sharedPreferences.getString(Constants.PREF_THEME, null)
     val dynamicColors get() = sharedPreferences.getBoolean(Constants.PREF_DYNAMIC_COLORS, true)
     val amoledTheme get() = sharedPreferences.getBoolean(Constants.PREF_AMOLED_THEME, false)
+    val spanCount get() = sharedPreferences.getInt(Constants.PREF_SPAN_NO, 2)
```

Comment:
Don't hardcode the value here, the dimens.xml contains a `library_columns` integer value which is used for different screen sizes as default.

https://github.com/jarnedemeulemeester/findroid/blob/5aad3781320ab7f989c54cd48a8638b3b88aa72c/core/src/main/res/values/dimens.xml#L9

https://github.com/jarnedemeulemeester/findroid/blob/5aad3781320ab7f989c54cd48a8638b3b88aa72c/core/src/main/res/values-sw720dp/dimens.xml#L4

https://github.com/jarnedemeulemeester/findroid/blob/5aad3781320ab7f989c54cd48a8638b3b88aa72c/core/src/main/res/values-sw600dp/dimens.xml#L6